apps/aega/test/aega_SUITE.erl

```diff
@@ -418,17 +434,23 @@ basic_minimum_fee(_Cfg) ->
                                          recipient_id => aeser_id:create(account, To),
                                          amount       => 4711,
                                          height       => Height,
+                                         fee          => 16700 * MinGP,
                                          nonce        => 0}),
     MinimumSpendFee = aetx:min_fee(DummyTx, Height, Protocol),
 
-    %% When the transaction is wrapped in a meta transaction, the size of the
-    %% inner transaction is paid for by the meta transaction.
+    %% Up until IRIS:
+    %%   When the transaction is wrapped in a meta transaction, the size of
+    %%   the inner transaction is paid for by the meta transaction.
+    %% From IRIS:
+    %%   The inner transaction should cover its own "size-cost"
     SizeFee = aetx:size(DummyTx) * aec_governance:byte_gas() * MinGP,
-    MinimumInnerFee = MinimumSpendFee - SizeFee,
+    MinimumInnerFee = case Protocol =< ?LIMA_PROTOCOL_VSN of
+                        true  -> MinimumSpendFee - SizeFee;
```

Comment:
This is not new code, but just wonder whether this could ever become a negative MinimumInnerFee?

Reply:
No it can't. Since `SizeFee` is part of `MinimumSpendFee`.